Resources/Prototypes/Floof/Entities/Mobs/NPCs/ertmothroach.yml

```
@@ -0,0 +1,36 @@
+- type: entity
+  name: ERT mothroach
+  parent: MobMothroach
+  id: MobERTMothroach
+  description: This lil guy seems ready for battle 
+  components:
+  - type: Sprite
+    sprite: Mobs/Animals/ertmothroach.rsi
+  - type: NpcFactionMember
+    factions:
+    - PetsNT
+  - type: PointLight
+    radius: 1.5
+    energy: 2.3
+    offset: 0, -1
+    color: green
```

```suggestion
  - type: AutoImplant
    implants:
    - LightImplant
```
This is just a suggestion but i think making it toggable using an implant is gud.